src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Nuget/NugetRestorer.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Threading;
+using NuGet.SolutionRestoreManager;
+using System;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Nuget
+{
+    using TIdentityDictionary = IImmutableDictionary<NamedIdentity, IComparable>;
```

Comment:
`TSomething` in .NET style generally means it's a type parameter. Is this a competing VS convention for aliases?